File: Plugins/Aspose_Pdf_Java_for_PHP/src/Aspose/Pdf/WorkingWithDocumentConversion/PdfToExcel.php
<?php

namespace Aspose\Pdf\WorkingWithDocumentConversion;

use com\aspose\pdf\Document as Document;
use com\aspose\pdf\ExcelSaveOptions as ExcelSaveOptions;

class PdfToExcel {

    public static function run($dataDir=null)
    {
        # Open the target document
        $pdf = new Document($dataDir . 'input1.pdf');

        # Instantiate ExcelSave Option object
        $excelsave = new ExcelSaveOptions();

        # Save the output to XLS format
        $pdf->save($dataDir . "Converted_Excel.xls", $excelsave);

        print "Document has been converted successfully" . PHP_EOL;
    }
}

File: Plugins/Aspose-Pdf-Java-for-Jython/asposepdf/WorkingWithDocumentConversion/PdfToExcel.py
from asposepdf import Settings
from com.aspose.pdf import Document
from com.aspose.pdf import ExcelSaveOptions

class PdfToExcel:

    def __init__(self):
        dataDir = Settings.dataDir + 'WorkingWithDocumentConversion/PdfToExcel/'
        
        # Open the target document
        pdf = Document(dataDir + 'input1.pdf')

        # Instantiate ExcelSave Option object
        excelsave = ExcelSaveOptions()

        # Save the output to XLS format
        pdf.save(dataDir + "Converted_Excel.xls", excelsave)

        print "Document has been converted successfully"
 
if __name__ == '__main__':        
    PdfToExcel()